docs/source/changelog.rst

```diff
@@ -1,7 +1,18 @@
 =========
 Changelog
 =========
+.. Future Release
```

Comment:
Uncomment this. Also if we are going to use the underline approach for the release version/date like we do in other repos, should we update the past entries so the formatting is consistent? Not a big deal either way though.

Reply:
Eh, let's leave it for now.
https://github.com/alteryx/nlp_primitives/issues/85

Reply:
Actually done.
https://github.com/alteryx/nlp_primitives/pull/84/commits/1deb85a86fce48b0a922260657d04c726968ab14